### PR TITLE
QUANTUM PEE

### PR DESCRIPTION
**Pull request by MeatyLearner**
they said it wasn't possible.... fianly........

### CODE REVIEW

**Review by C-rank — APPROVED** [review]
looks good to me chief 😎😎😎

File: ew/cmd/mutation/mutationcmds.py [hunk 1]
```diff
@@ -407,6 +407,8 @@ async def piss(cmd):
                     return await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
 
                 response = "You piss HARD and FAST right onto {}!!".format(target_member.display_name)
+            elif ewcfg.mutation_id_quantumlegs in mutations:
```
Comment (aristotlepenguin) [inline ew/cmd/mutation/mutationcmds.py:410]: dude what if you made it so quantum legs spent its charge if you quantum pissed

Comment (MeatyLearner) [reply ew/cmd/mutation/mutationcmds.py:410]: You're right. I let my biases get in the way of game balance. I'll add the cooldown

File: ew/cmd/wep/wepcmds.py [hunk 2]
```diff
@@ -1001,7 +1001,7 @@ async def annoint(cmd):
 
     # Grab the wanted name
     if cmd.tokens_count < 2:
-        annoint_name = weapon_item.item_props.get("weapon_name")
+        annoint_name = weapon_item.name
```
Comment (C-rank) [inline ew/cmd/wep/wepcmds.py:1004]: looks good broski but this was already covered in another (now merged) PR

File: ew/cmd/fish/fishcmds.py [hunk 3]
```diff
@@ -434,6 +434,7 @@ async def cast(cmd):
 async def reel(cmd):
     user_data = EwUser(member=cmd.message.author)
     resp_ctn = fe_utils.EwResponseContainer(client=cmd.client, id_server=cmd.guild.id)
+    responses = []
```
Comment (C-rank) [inline ew/cmd/fish/fishcmds.py:437]: looks good broski but this was already covered in another (now merged) PR

File: client.py [hunk 4]
```diff
@@ -911,7 +911,7 @@ async def on_message(message):
 
             response = "ENDLESS WAR completely and utterly obliterates {} with a bone-hurting beam.".format(message.author.display_name).replace("@", "\{at\}")
             return await fe_utils.send_message(client, message.channel, response)
-        if str(message.channel) in ["nurses-office", "suggestion-box", "detention-center", "community-service", "playground", "graffiti-wall", "post-slime-drip", "outside-the-lunchroom", "outside-the-lunchrooom"] or message.channel.type in [discord.ChannelType.public_thread, discord.ChannelType.private_thread]:
+        if str(message.channel) in ["nurses-office", "suggestion-box", "detention-center", "community-service", "playground", "graffiti-wall", "post-slime-drip", "outside-the-lunchroom", "outside-the-lunchrooom", "outside-the-lunchroooom"] or message.channel.type in [discord.ChannelType.public_thread, discord.ChannelType.private_thread]:
```
Comment (C-rank) [inline client.py:914]: looks good broski but this was already covered in another (now merged) PR

**Review by C-rank — COMMENTED** [review]
looks good broski but a few different things were already covered in another (now merged) PR

plus i checked out the code on my test bot and everything works good 😎

**Pull request merged**